clang/lib/Driver/Driver.cpp

```diff
@@ -169,6 +169,17 @@ getHIPOffloadTargetTriple(const Driver &D, const ArgList &Args) {
   return std::nullopt;
 }
 
+
+static std::optional<llvm::Triple>
+getINTELOffloadTargetTriple(const Driver &D, const ArgList &Args,
```

Comment:
The name doesn't seem to fit the triple being returned.  The value here is a `spirv` value, so is this more of a 'default SYCL JIT' triple?